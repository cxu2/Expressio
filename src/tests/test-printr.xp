~~ comment
main : -> int {
 
  regexp zero;
  regexp one;
  regexp x;
  regexp y;
  regexp z;

  regexp t;
  regexp w;
  regexp s;

  zero  = {.};
  one = {{.}};
  printr(zero);
  printr(one);

  x = lit 'c';
  y = lit 'a';
  z = x ^ y;

  t = x | z;
  w = y & t;
  s = w **;

  printr(s);

  return (0);
}
