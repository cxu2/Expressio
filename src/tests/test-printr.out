{.}
{{.}}
( ( a & ( c | ( c ^ a ) ) )*  &  'c )